### PR TITLE
diff: escape column name

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 

if the column name is keyword in mysql/tidb, will return error when select data.

### What is changed and how it works?

escape column name

### Check List 

Tests 

 - Unit test
 - Integration test

### CODE REVIEW

**Comment by WangXiangUSTC** [comment]
@lichunzhu @csuzhangxc PTAL

**Review by lichunzhu — COMMENTED** [review]
LGTM

File: pkg/diff/diff.go [hunk 1]
```diff
@@ -829,6 +829,10 @@ func getChunkRows(ctx context.Context, db *sql.DB, schema, table string, tableIn
 		collation = fmt.Sprintf(" COLLATE \"%s\"", collation)
 	}
 
+	for i, key := range orderKeys {
+		orderKeys[i] = dbutil.ColumnName(key)
```
Comment (csuzhangxc) [inline pkg/diff/diff.go:833]: Do we need to convert for `schema`, `table` and `columns` bellow?

**Review by csuzhangxc — COMMENTED** [review]
rest LGTM

**Review by csuzhangxc — DISMISSED** [review]
LGTM

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**